Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:boron
+
+# Create app directory
+RUN mkdir -p /usr/src/dc-campaign-finance-watch
+WORKDIR /usr/src/dc-campaign-finance-watch
+
+# Install app dependencies
+COPY package.json /usr/src/dc-campaign-finance-watch/
+RUN npm install
+
+# Bundle app source
+COPY . /usr/src/dc-campaign-finance-watch
```

Comment:
It's copying the js files from the current repo dir into the docker image so when the server starts it can use the appropriate files.